include/bpf_endian.h

```diff
@@ -4,13 +4,78 @@
 
 #include <stdint.h>
 
-inline __attribute__((always_inline)) uint16_t
-bpf_ntohs(uint16_t us)
-{
-    return us << 8 | us >> 8;
-}
+/*
```

Comment:
It was just pointed out to me that it looks like this PR copies code from https://github.com/torvalds/linux/blob/master/tools/lib/bpf/bpf_endian.h
To adhere to both this project's Contributor License Agreement and the license in that other file:
if code was copied from there, it's important to acknowledge the source and the SPDX identifier of the source.
Hence assuming this was copied, please add a comment in the code acknowledging the source URL and citing
its SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause).

Reply:
Does my new commit meet the requirements?